pkg/base-dev/sur/hood.hoon

```diff
@@ -108,6 +111,64 @@
       leaf/"kids desk:        {?~(sink <~> ?~(kid.u.sink <~> <u.kid.u.sink>))}"
       leaf/"pending updates:  {<`(list [@tas @ud])`~(tap in wic.dek)>}"
   ==
+
+++  report-vat-short
+  |=  $:  $:  tyr=rock:tire  =cone  sor=(map desk [ship desk])
```

Comment:
style:  I'd put each of these on their own line, like this:
```hoon
$:  tyr=rock:tire
    =cone
    sor=(map desk [ship desk])
    zyn=(map [desk ship desk] sync-state)
==
```

Reply:
Sometimes when making a large noun, such as a move to emit, we will have multiple entries in a `:*` on the same line, but the standard for `$:` is to put each item on its own line.